From a537e67a5eb71ca63b6d09761c65f0aa47e9fe10 Mon Sep 17 00:00:00 2001
From: Nolen Johnson <johnsonnolen@gmail.com>
Date: Wed, 27 Dec 2017 17:07:38 -0500
Subject: [PATCH 041/249] wiki: Change fugu vendor

* Long Vendor is Google (who sold it as a Nexus device).
* Short Vendor is Asus (the OEM who actaully manufactured it).

Change-Id: I38ab4b9993e7c88d08132a86e37c2c19358b959a
---
 _data/devices/fugu.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/_data/devices/fugu.yml b/_data/devices/fugu.yml
index c5d001d..bca457e 100644
--- a/_data/devices/fugu.yml
+++ b/_data/devices/fugu.yml
@@ -38,7 +38,7 @@ soc: Intel Atom x86_64 (Moorefield)
 storage: 8 GB eMMC
 tree: android_device_asus_fugu
 type: Set top box
-vendor: Asus
+vendor: Google
 vendor_short: asus
 versions: [14.1]
 width:
-- 
2.17.1

